docs/Guide.Jest.md

```diff
@@ -0,0 +1,57 @@
+# Jest as Test Runner
+
+## Usage
+
+### 0. Use the [Getting Started](Introduction.GettingStarted.md) Guide to set up detox
+
+Except that you need to skip the install mocha step.
+
+### 1. Install Jest
+
+```sh
+npm install --save-dev jest
+```
+
+### 2. Remove mocha specific files
+
+You should remove `e2e/mocha.opts`, you no longer need it.
+
+### 3. Write a detox setup file
+
+```js
+const detox = require('detox');
+const config = require('../package.json').detox;
+
+// Set the default timeout
+jasmine.DEFAULT_TIMEOUT_INTERVAL = 120000;
+
+beforeAll(async () => {
+  await detox.init(config);
+});
+
+afterAll(async () => {
+  await detox.cleanup();
+});
+
+// optional, you may remove this part
+beforeEach(async () => {
+  await device.reloadReactNative();
+});
+```
+
+### 4. Run jest
+
+Add this part to your `package.json`:
+```json
+"scripts": {
+    "test:e2e": "jest e2e --setupTestFrameworkScriptFile=./jest/setup-e2e-tests.js"
```

Comment:
I like how you can define jest config in your package.json so you don't have to pass in `--setupTestFrameworkScriptFile`. Just making a note. either way is fine. 

```
"jest": {
    "bail": true,
    "setupTestFrameworkScriptFile": "<rootDir>/jest/setup.js"
  },
```

Reply:
This is intentional because we wanted to make sure that they run side by side with your normal unit tests

Reply:
sure. either way is fine. you can push that logic in setup file too: https://github.com/grabbou/react-native-detox-example/blob/master/jest/setup.js#L10-L14